data/_models/shared/Collections.csv

```diff
@@ -1,53 +1,51 @@
-Collections,,other organisations,text,,,,,,,,dcterms:contributor,List the names of any other organisations that are not listed and contributed to this resource,EMA,['Network'].includes(collectionType?.name)
-Collections,,description,text,,,,,,,,dcterms:description,Short description,"DataCatalogueFlat,EMA",
+Collections,,description,text,,,,,,,,dct:description,Short description,"DataCatalogueFlat,EMA",
 Collections,,data collection description,text,,,,,,,,,Describe the process of collection and recording of data.,"DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
 Collections,,date established,date,,,,,,,,,"Date when the data source was first established. If the exact day of the month is not known, please enter 15. If the exact month is not known then please enter 15/06","DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
 Collections,,start data collection,date,,,,,,,,dcat:startDate,"The date when data started to be collected or extracted. If the exact day of the month is not known, please enter 15. If the exact month is not known then please enter 15/06","DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
 Collections,,end data collection,date,,,,,,,,dcat:endDate,"If data collection in the data source has ceased, on what date did new records last enter the data source?. If the exact day of the month is not known, please enter 15. If the exact month is not known then please enter 15/06","DataCatalogueFlat,EMA","['Databank', 'Data source'].includes(collectionType?.name)"
 Collections,,time span description,text,,,,,,,,,Description of time span,DataCatalogueFlat,"['Databank', 'Data source'].includes(collectionType?.name)"
+Collections,,external identifiers,refback,,,,External identifiers,,resource,,,"External identifier(s) for this resource (e.g. EUPASS number, UMCG register number)","DataCatalogueFlat,EMA","['Cohort', 'Databank', 'Data source', 'Model', 'Study'].includes(collectionType?.name)"
 Collections,,status,ontology,,,CatalogueOntologies,Study status,,,,,Status of the study,"DataCatalogueFlat,EMA",['Study'].includes(collectionType?.name)
 Collections,,release frequency,int,,,,,,,,,Refreshing rate (in months),"DataCatalogueFlat,EMA",['Model'].includes(collectionType?.name)
-Collections,,contact email,,,,,,,,,dcat:contactPoint,Contact e-mail address for this cohort,DataCatalogueFlat,['Cohort'].includes(collectionType?.name)
-Collections,,cohort type,ontology_array,,,CatalogueOntologies,Resource types,,,,dcterms:type,"Type of resource, e.g. registry, cohort, biobank",DataCatalogueFlat,['Cohort'].includes(collectionType?.name)
+Collections,,contact email,,,,,,,,,dcat:contactPoint,Contact e-mail address for this collection,DataCatalogueFlat,['Cohort'].includes(collectionType?.name)
```

Comment:
dcat:contacPoint needs vcard:Kind. I do not know whether we can annotate it like this. Can we annotate from the Contacts table and bring stuff together in the RDF?